test/util.ts

```diff
@@ -1,7 +1,7 @@
-import {randomBytes} from "../src/helpers/index.js";
+import crypto from "crypto";
```

Comment:
Would be good to import native node modules with `node:` prefix

Reply:
Unfortunately due to `webpack` it cannot be done. The following error is thrown:

```
Module build failed: UnhandledSchemeError: Reading from "node:crypto" is not handled by plugins (Unhandled scheme).
Webpack supports "data:" and "file:" URIs by default.
```

That was originally part of my original [commit](https://github.com/ChainSafe/bls/pull/151/commits/aa152de064d5e64905b541e0756cc321a9873f35)

Probably the best way to address that longer term is to move to vitest.